cdap-storage-ext-spanner/src/test/java/io/cdap/cdap/storage/spanner/SpannerStructuredTableTest.java

```diff
@@ -50,10 +50,10 @@ public class SpannerStructuredTableTest extends StructuredTableTest {
 
   @BeforeClass
   public static void createSpannerStorageProvider() throws Exception {
-    String project = System.getProperty("gcp.project");
-    String instance = System.getProperty("gcp.spanner.instance");
-    String database = System.getProperty("gcp.spanner.database");
-    String credentialsPath = System.getProperty("gcp.credentials.path");
+    String project = "skilled-mark-331618";
```

Comment:
pls revert these changes, currently SpannerTable tests are running in real env so they're skipped in unit tests